<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinarySearcherPlugin::DialogAsciiString</class>
 <widget class="QDialog" name="BinarySearcherPlugin::DialogAsciiString">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find ASCII String</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ASCII String</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtAscii">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="chkCaseSensitive">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Case Sensitive</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtAscii</tabstop>
  <tabstop>chkCaseSensitive</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BinarySearcherPlugin::DialogAsciiString</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BinarySearcherPlugin::DialogAsciiString</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
